### PR TITLE
app-mobilephone/pinephone-keyboard: new package, version 1.2

### PR DESCRIPTION
**Pull request by ulziibuyan**
The package provides tools, firmware and daemon for the PinePhone Keyboard. It has been tested on PinePhone 1.2a with the PinePhone Keyboard.

@Leo3418 This is tagged with version 1.2.

Signed-off-by: ULZIIBUYAN ERDENEBILEG <lzijbuan@gmail.com>

### CODE REVIEW

**Comment by ulziibuyan** [comment]
@graaff Would you be kind enough to add hacktoberfest-label to this PR? I would love this contribution to count towards Hacktoberfest 2022. I became Gentoo's GURU project contributor few hours ago. I will close this PR and commit the new package directly.

File: app-mobilephone/pinephone-keyboard/pinephone-keyboard-1.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		Restart=on-failure
+		RestartSec=10
+		KillMode=process
+
+		[Install]
+		WantedBy=multi-user.target
+		EOF
+
+	default
+}
```
Comment (scardracs) [inline app-mobilephone/pinephone-keyboard/pinephone-keyboard-1.2.ebuild:34]: Why not creating a pinephone-keyboard.service under files/ ?

**Comment by scardracs** [comment]
Where are metadata.xml and Manifest?

**Comment by rahul-singh-sandhu** [comment]
@ulziibuyan You can generate the manifest with **pkgdev manifest** . Look at another metadata.xml file in the repo as an example on how to write one (its fairly simple).

**Pull request closed without merging**